src/Google/Ads/GoogleAds/Lib/V1/LogMessageFormatter.php

```diff
@@ -40,6 +41,26 @@ public function __construct($statusMetadataExtractor = null)
         $this->statusMetadataExtractor = $statusMetadataExtractor ?: new StatusMetadataExtractor();
     }
 
+    /**
+     * Returns the customer ID information of the provided request.
+     *
+     * @param Message $request the request to get its customer ID
+     * @return string the customer ID information
+     */
+    private static function getCustomerIdInfo(Message $request): string
+    {
+        if (method_exists($request, 'getCustomerId')) {
```

Comment:
Could you add a quick comment explaining that it is covering all mutate requests?

Reply:
It's not always the mutate requests. Search requests also are included here.
Listing all methods (which can be added or deleted later) will cause maintenance problems in the future.
I think Python and Perl versions are also fine. It's clear from the code itself.